rules/properties-order/checkNode.js

```diff
@@ -91,7 +91,9 @@ module.exports = function checkNode(node, sharedInfo, originalNode) {
 			return;
 		}
 
-		checkEmptyLineBefore(previousNodeData, nodeData, sharedInfo);
+		const propsCount = node.nodes.filter(item => utils.isProperty(item));
```

Comment:
Sorry, I forgot `.length`.

```js
const propsCount = node.nodes.filter(item => utils.isProperty(item)).length;
```

And put it outside `allNodesData.forEach()` cycle, because there is no need to calculate it for every child node.